x-pack/plugins/infra/public/pages/metrics/hosts/types.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export type TabId = 'alerts' | 'metrics';
```

Comment:
If we decide to apply the suggestion about `tabIds`, we could possibly remove this type.